openapi/components/codeSamples/economy.yaml

```diff
@@ -40,3 +40,39 @@
       source: >-
         curl -X GET "https://vrchat.com/api/1/licenseGroups/{licenseGroupId}" \
              -b "auth={authCookie}"
+'/listing/{productId}':
+  get:
+    - lang: cURL
+      source: >-
+        curl -X GET "https://vrchat.com/api/1/listing/{productId}" \
+             -b "auth={authCookie}"
+'/user/{userId}/listings':
+  get:
+    - lang: cURL
+      source: >-
+        curl -X GET "https://vrchat.com/api/1/user/{userId}/listings?hydrated=true" \
```

Comment:
``hydrated`` is not defined in the spec, did you mean `hydrate`?

Reply:
yup
i guess this is the result of a pr made at 2am